website/src/components/MiniBlog/index.tsx

```diff
@@ -214,13 +214,13 @@ const MiniBlog: React.FC = () => {
           >
             {filteredData.map((elm: any) => {
               return (
-                <div>
+                <div className={classes.cardWrapper}>
                   <Card key={elm.id} className={classes.cardRoot}>
                     <CardMedia
                       className={classes.media}
                       image={`/Images/blog/${elm.slug}.png`}
                     />
-                    <CardContent>
+                    <CardContent className={classes.cardContent}>
```

Comment:
Can we add these changes for the blog slider as well? website/src/components/BlogsSlider/index.tsx

Reply:
its already been fixed @Pallavi-PH 
---

<img width="1409" alt="Screenshot 2021-06-15 at 9 20 47 AM" src="https://user-images.githubusercontent.com/6524419/121990213-061bdd80-cdbb-11eb-8b84-b14a3ed7ea57.png">